### ISSUE TITLE
Clash with `nvim-ufo`

### ISSUE THREAD

**Issue by ranjithshegde** [folke/noice.nvim#91]
**Describe the bug**
Noice breaks either when calling `require("ufo").closeAllFolds()` manually or by resourcing a file with closed folds.

I get the error message `vim.notify has been overwritten by another plugin?` and after that the noice cmdline remains frozen.. 
Calls to cmdline are handled by the default cmdline thereafter
`

**Which version of Neovim are you using?**
NVIM v0.9.0-dev-a4c9cca

**To Reproduce**
1. Install noice & nvim-ufo
2. Use any fold method (or none)
3. Call `require("ufo").closeAllFolds()`

**Expected Behavior**
No error.

**Screenshots**
Error line
![error](https://user-images.githubusercontent.com/10258296/197176994-576c01fc-ffb3-46ef-b35e-40af0ef709ea.png)

Frozen line
![frozen](https://user-images.githubusercontent.com/10258296/197177010-7e275cc3-21d8-43a4-9140-73f185292d38.png)

**Noice Log**
[noice.log](https://github.com/folke/noice.nvim/files/9838257/noice.log)

**Comment by ranjithshegde** [comment]
Trying against the latest commit produces different results.

This time calling `require("ufo").closeAllFolds()` no longer crashes `noice`. It simply freezes the current `cmdline` until `:` or `/` is pressed next. And also produces no logs

**Comment by folke** [comment]
Should be fixed now. Let me know if you have any more issues!

**Comment by ranjithshegde** [comment]
Works just fine now! There is almost one second delay in hiding the `cmdline`,  only on the problematic call.. but I dont really notice it if I am not looking for it!

Thank you for the fix, and this amazing plugin

**Comment by folke** [comment]
Yep, that's normal. It seems that ufo is doing something, that prevents me from closing the window. I get E565 errors.

The fix was to retry and defer the closing on E565 errors by 100ms.

**Comment by ranjithshegde** [comment]
I see. That makes sense.

Thanks again
